tutor/src/components/task-teacher-review/exercise.cjsx

```diff
@@ -16,7 +16,7 @@ TaskTeacherReviewQuestion = React.createClass
 
   onChangeAnswerAttempt: (answer) ->
     # TODO show cannot change answer message here
-    console.log('You cannot change an answer on a problem you\'ve reviewed.', 'TODO: show warning in ui.')
+    alert('You cannot change an answer on a problem you\'ve reviewed.')
 
```

Comment:
the alert looks scary...could we take out both the alert and console log for now, `return null` from this, and update the TODOs to say, `clean up onChangeAnswerAttempt`?  That feature never got fully implemented and is not relevant anymore, so we can remove some dead code around that